### PR TITLE
Update coding standards

### PR DESCRIPTION
**Pull request by mikkoaf**
This PR updates Seravo-plugin coding standards.

### CODE REVIEW

File: lib/helpers.php [hunk 1]
```diff
@@ -31,6 +31,15 @@ public static function is_staging() {
       return ( getenv('WP_ENV') && getenv('WP_ENV') === 'staging' );
     }
 
+    public static function seravo_plugin_version() {
+      if ( is_admin() ) {
+        $var = '1.0';
+        $var = get_plugin_data( WP_CONTENT_DIR . '/mu-plugins/seravo-plugin/seravo-plugin.php' );
+        $var = $var['Version'];
+        return $var;
```
Comment (l3ku) [inline lib/helpers.php]: Can you explain in words what is going on here? The same variable is assigned multiple times for some reason.

File: modules/backups.php [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 namespace Seravo;
 
+use Seravo\Helpers;
```
Comment (l3ku) [inline modules/backups.php]: If you're inside the `Seravo` namespace, to my understanding you could just use the `Helpers` class directly without using `Seravo\Helpers` inside `Seravo`.

File: phpcs.xml [hunk 3]
```diff
@@ -52,7 +52,9 @@
     assessed separately. Also we can't always use nonces e.g. for APIs... -->
     <exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification" />
     <exclude name="WordPress.XSS.EscapeOutput" />
-
+    <exclude name="WordPress.Security.EscapeOutput.UnsafePrintingFunction" />
+    <exclude name="WordPress.Security.EscapeOutput.OutputNotEscaped" />
+    <exclude name="WordPress.Security.NonceVerification.NoNonceVerification" />
```
Comment (l3ku) [inline phpcs.xml]: Please provide an explanation for each of these as to why they are excluded (see other excludes for example). I bet that excluding them is reasonable but it is always good to provide some reference.

**Comment by l3ku** [comment]
Also as a notice, note that your branch is named `origin/update-coding-standards`, if not intended, this might cause some confusion as the default remote is named `origin` in git :).

File: lib/helpers.php (hunk 1 above)
Comment (elguitar) [reply lib/helpers.php]: I apologize for jumping in like this but I would suggest that the $var should be named in a more descriptive way.

File: phpcs.xml (hunk 3 above)
Comment (mikkoaf) [reply phpcs.xml]: These exclusions are related to the comment on rows 51-52.

File: lib/helpers.php (hunk 1 above)
Comment (l3ku) [reply lib/helpers.php]: I agree. No need to apologize, the more eyes we get on code reviews the better :)

File: phpcs.xml (hunk 3 above)
Comment (l3ku) [reply phpcs.xml]: Ok 👍

File: lib/helpers.php (hunk 1 above)
Comment (mikkoaf) [reply lib/helpers.php]: @elguitar thanks for the input! I'll keep this in mind - even if this variable is used for only a few rows, is better to describe it well for better readability.

File: modules/backups.php (hunk 2 above)
Comment (mikkoaf) [reply modules/backups.php]: For some reason I could not replicate what you described.

**Comment by mikkoaf** [comment]
The requested changes have been made and the branch has been rebased.

**Comment by l3ku** [comment]
The changes made to the static asset query strings are not most likely correct:
screen shot 2018-08-01 at 16 09 43

Also, you can remove the lines `use Seravo\Helpers;` when inside the`Seravo` namespace (tested and works).

**Comment by mikkoaf** [comment]
Latest changes address the requested changes.

File: lib/helpers.php [hunk 4]
```diff
@@ -31,6 +31,12 @@ public static function is_staging() {
       return ( getenv('WP_ENV') && getenv('WP_ENV') === 'staging' );
     }
 
+    public static function seravo_plugin_version() {
+      if ( is_admin() ) {
```
Comment (l3ku) [inline lib/helpers.php]: I was already about to merge, but could you explain why this function only returns something when the user is in WP admin? We load e.g. `instance-switcher.js` and `instance-switcher.css` in the frontend side, and this results in the version query strings missing as the function returns `null`.

**Pull request merged**